tests/baselines/reference/noExcessPropertyCheckWithGenericMappedType1.errors.txt

```diff
@@ -0,0 +1,58 @@
+noExcessPropertyCheckWithGenericMappedType1.ts(9,9): error TS2322: Type '{}' is not assignable to type 'Target'.
+noExcessPropertyCheckWithGenericMappedType1.ts(10,9): error TS2322: Type '{ abc: string; }' is not assignable to type 'Target'.
+noExcessPropertyCheckWithGenericMappedType1.ts(13,9): error TS2322: Type '{ other: number; }' is not assignable to type 'Target'.
+noExcessPropertyCheckWithGenericMappedType1.ts(20,9): error TS2322: Type '{ abc: number; }' is not assignable to type 'Target'.
+noExcessPropertyCheckWithGenericMappedType1.ts(21,9): error TS2322: Type '{}' is not assignable to type 'Target'.
+noExcessPropertyCheckWithGenericMappedType1.ts(22,9): error TS2322: Type '{ abc: string; }' is not assignable to type 'Target'.
+noExcessPropertyCheckWithGenericMappedType1.ts(24,9): error TS2322: Type '{ abc: number; other: number; }' is not assignable to type 'Target'.
+noExcessPropertyCheckWithGenericMappedType1.ts(25,9): error TS2322: Type '{ other: number; }' is not assignable to type 'Target'.
+noExcessPropertyCheckWithGenericMappedType1.ts(26,9): error TS2322: Type '{ abc: number; other: string; }' is not assignable to type 'Target'.
+
+
+==== noExcessPropertyCheckWithGenericMappedType1.ts (9 errors) ====
+    // https://github.com/microsoft/TypeScript/issues/56391
+    
+    type S = { x: "abc" };
+    
+    function f<T extends S>() {
+      type Target = { [k in T["x"]]: number };
+    
+      const x: Target = { abc: 1 };
+      const y: Target = {};
+            ~
+!!! error TS2322: Type '{}' is not assignable to type 'Target'.
+      const z: Target = { abc: "" };
+            ~
+!!! error TS2322: Type '{ abc: string; }' is not assignable to type 'Target'.
+    
+      const a: Target = { abc: 1, other: 2 };

```

Comment:
The `getConstraintOfType(getConstraintTypeFromMappedType(type))` is a string literal. So this one (just like `c` here) could, technically, report an EPC. It feels to me that implementing it might not be worth it, but I can certainly do it if requested.